packages/@okta/vuepress-site/docs/reference/api/domains/index.md

```diff
@@ -185,26 +186,27 @@ curl -v -X POST \
 }
 ```
 
-### Create Certificate
+### Create certificate
 
 <ApiOperation method="put" url="/api/v1/domains/${id}/certificate" />
 
-Creates the Certificate for the Domain
+Creates the certificate for the Domain.
+If the certificateSourceType in the [Domain](#domain-object) is `OKTA_MANAGED`, it will now become `MANUAL` and Okta will no longer manage and renew certificates for this domain since a user-managed certificate has been provided.
```

Comment:
```suggestion
If the `certificateSourceType` in the [Domain](#domain-object) is `OKTA_MANAGED`, it becomes `MANUAL` and Okta no longer manages and renews certificates for this domain since a user-managed certificate has been provided.
```

Reply:
Sure.  If you configure a domain with a certificateSourceType of MANUAL, you need to give us a cert, because you are managing certs manually yourself.  If you configure a domain with certificateSourceType as OKTA_MANAGED, Okta will obtain and manage the certificate for you. If chose to have Okta manage the domain certificate for you but you come along at some later time and give us a valid certificate, we can't manage the domain certificate anymore because you gave us one.  It is like when you have cruise control on in your car but hit the brakes or gas - the cruise control automatically turns off because you have demanded to take over control by your actions and the car has complied.

Reply:
gotcha! thank you - makes perfect sense.